.github/workflows/create-release.yaml

```
@@ -1,53 +1,280 @@
 name: "Create release"
-run-name: "Create release ${{ inputs.release_version }}"
-
-permissions:
-  contents: write
+run-name: "Create release ${{ inputs.name }} force: ${{ inputs.force }} dry-run: ${{ inputs.dry-run }}"
 
 on:
   workflow_dispatch:
     inputs:
-      release_version:
-        description: "Release version (i.e. x.y.z)"
+      name: # release tag
+        description: "Release tag (i.e. x.y.z)"
```

```suggestion
        description: "Release tag (in the format x.y.z)"
```
that were so many single letters with periods in between! :D